common/test/acceptance/tests/lms/test_learner_profile.py

```diff
@@ -332,11 +345,17 @@ def test_birth_year_not_set(self):
 
         Given that I am a registered user.
         And birth year is not set for the user.
-        And i visit my profile page.
-        Then i should see message `Your profile is disabled because you haven't filled in your Year of Birth.`
+        And I visit my profile page.
+        Then I should see a message that the profile is private until the year of birth is set.
         """
+        username, user_id = self.log_in_as_unique_user()
```

Comment:
Question: how does this work since birth year is set by default now in auto auth code?